### ISSUE TITLE
Cannot reproduce the results on AffectNet

### ISSUE THREAD

**Issue by zyh-uaiaaaa** [Talented-Q/POSTER_V2#12]
Hi,

Could you please upload the pre-processing code of AffectNet? It seems that I could not reproduce the results even on my own AffectNet test set using your uploaded trained model. Additionally, training from scratch on my AffectNet dataset cannot achieve over 67% on 7 classes or over 63% on 8 classes. Thanks for your help.

**Comment by Talented-Q** [comment]
![J04VDBJ8R3%WJG10HYNN9}V](https://github.com/Talented-Q/POSTER_V2/assets/79841528/c25f2561-b848-4466-b8e9-6ef36b521ce5)

Thank you for your interest in this paper. sampler=ImbalancedDatasetSampler(train_dataset)。
The Affectnet dataset should be oversampled during training. sampler=ImbalancedDatasetSampler(train_dataset) should be added to train_loader. We apologize that we did not point out that we had given you the wrong impression. We will make changes to the above issues in the coming work.

**Comment by romanceyu** [comment]
> 你好
> 
> 您能否上传 AffectNet 的预处理代码？似乎即使在我自己的 AffectNet 测试集上使用您上传的训练模型也能重现结果。此外，在我的 AffectNet 数据集上从头开始训练无法在 7 个类上达到超过 67% 或在 8 个类上达到超过 63%。感谢您的帮助。

Can you share the AffectNet DataSet with me ?
